frontend/src/models.ts

```diff
@@ -16,6 +16,21 @@ export type SongEntry = {
   is_flagged: boolean;
 };
 
+type WishlistSong = {
+  id: number;
+  artist: string;
+  title: string;
+  created_at: string;
+  updated_at: string;
+};
+
+export type WishlistSongs = {
```

Comment:
There's a pagination object you can use for this, check out `DjangoPaginatedResponse` (and it's uses). So you can delete `WishlistSongs` and use that instead!